### ISSUE TITLE
transfer learning의 accuracy관련 문의사항입니다.

### ISSUE THREAD

**Issue by yeahoon-k** [public-ai/ALAI_Homework#77]
**1. weight초기화 방법에 관한 질문입니다.** 

weight의 transfer 방법을 두개로 나누어 수행했습니다.

- 첫번째는 saver.restore를 사용하여 restore했으며, 이때 Momentum optimizer를 따로 초기화해주어야 했습니다.

- 두번째는 weight를 numpy로 저장하여 Variable의 initializer를사용했습니다.
저는 tf.constant_initializer(weight)의 형식으로 initialize를 지정해주었습니다 (variabel내부에 constant를 초기화 하기위해). 다음으로 sess(tf.global_variable_initializer())를 통해 초기화하여 
saver.restore를 하지 않고 수행했습니다. Momentum initialize문제도 발생하지 않았습니다.

<img width="686" alt="init_kernel" src="https://user-images.githubusercontent.com/48887480/59666458-d186e280-91ef-11e9-8805-0e52a0c08aff.PNG">

두 방법 모두 정상적으로 수행되나, 두번째 방법이 맞는 방법인지 궁금합니다..

**2. 다음 질문은 학습 도중에 생긴 문제입니다.**

총 4번의 transfer learning을 수행했으며 각 learning rate은 이전 값의 1/10으로 감소된 값으로 수행했습니다.

아래 그림은 각 transfer learning 마다 최종 trade off 구간의 val loss와 acc입니다.

3번째 transfer learning이후로는 시작하자 마자 loss가 올라가면서 학습이 안되는 상황이 발생했습니다.

아무래도 해당 parameter를 갖는 model로는 이정도 acc가 한계인 것같습니다..

**이런 경우는 convolutioin layer를 깊게하거나 fc layer를 깊게하는 등의 model자체를 변경해줘야 하는 것인지 궁금합니다..**

<img width="400" alt="Loss의 변화" src="https://user-images.githubusercontent.com/48887480/59664816-b8306700-91ec-11e9-94ef-1a753acb24b8.PNG">

감사합니다.

**Comment by pai-plznw4me** [comment]
아주 잘 수행하셨습니다! learning rate 을 잘 조절하면 85% 까지도 올라갈수 있습니다. 
다음 숙제는 이제 학습한 weights 을 이용해 convolution 을깊게 하는것 입니다. 
숙제는 따로 공지하겠습니다.
